env0/resource_sshkey_test.go

```diff
@@ -0,0 +1,41 @@
+package env0
+
+import (
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"testing"
+)
+
+func TestUnitSshKeyResource(t *testing.T) {
+	resourceType := "env0_ssh_key"
+	resourceName := "test"
+	resourceFullName := resourceAccessor(resourceType, resourceName)
+	sshKey := client.SshKey{
+		Id:    "id0",
+		Name:  "name0",
+		Value: "Key🔑",
+	}
+
+	testCase := resource.TestCase{
+		ProviderFactories: testUnitProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: resourceConfigCreate(resourceType, resourceName, map[string]string{
+					"name":  sshKey.Name,
+					"value": sshKey.Value,
+				}),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceFullName, "id", sshKey.Id),
+					resource.TestCheckResourceAttr(resourceFullName, "name", sshKey.Name),
+					resource.TestCheckResourceAttr(resourceFullName, "value", sshKey.Value),
```

Comment:
@razbensimon So the decrypted ssh key exists on the returned object? 🤔

Reply:
this is the `resource`,  not the `data` tests

Reply:
Yeah but still, I get that it should be a part of the `SshKeyCreatePayload` but I'm not sure it should be a part of the ssh key schema because nobody uses it and it's a secret value. WDYT?

Reply:
put sensitive for now, and lets talk with the team

Reply:
7887c41